### ISSUE TITLE
Continue walking despite error?

### ISSUE THREAD

**Issue by hydrabolt** [jprichardson/node-klaw#1]
Is there a way to get klaw to continue walking through a directory even if an error occurs by just ignoring the file/directory it encountered the error on? I'm wanting to walk through an entire `/home/` but when I encounter a permission error the walk ends without emitting `end`. Thanks

**Comment by jprichardson** [comment]
Is it a crash? Could you post the stack trace output? Presumably you're listening for the `error` event, right?

**Comment by hydrabolt** [comment]
Yep, listening for the error event, I also tried the examples but the same occurs. Maybe it's because I'm on node 5.0.0?

```
{ [Error: EACCES: permission denied, scandir '/home/hydrabolt/Sync']
  errno: -13,
  code: 'EACCES',
  syscall: 'scandir',
  path: '/home/hydrabolt/Sync' }
```

I fixed it by modifying some of the code, I made a PR: https://github.com/jprichardson/node-klaw/pull/2

**Comment by jprichardson** [comment]
Published `klaw@1.1.3`. Presumably you're using `fs-extra`? If so, just reinstall fs-extra and you should have the latest `klaw`. Thanks for your help!

**Comment by hydrabolt** [comment]
Yes I am! Thanks!
